src/frontend/src/views/Upload/components/Metadata/index.tsx

```diff
@@ -46,6 +48,22 @@ export default function Metadata({
     return createStringToLocationFinder(namedLocations);
   }, [namedLocations]);
 
+  const handleRowMetadata_ = (id: string, sampleMetadata: MetadataType) => {
+    setMetadata((prevMetadata) => {
+      return { ...prevMetadata, [id]: sampleMetadata };
+    });
+  };
+
+  const handleRowMetadata = useCallback(handleRowMetadata_, [setMetadata]);
+
+  const applyToAllColumn_ = (fieldKey: keyof MetadataType, value: unknown) => {
```

Comment:
same type comment here

Reply:
this was mostly borrowed from the upload flow table, fixing the types isn't exactly straightforward so i added a TODO and hope to address it later.

Reply:
i moved applyToAllColumn and handleRowMetadata up to the parent components of webformtable to better handle tracking the changes to metadata